docs/deployment.rst

```diff
@@ -156,5 +160,5 @@ and no other interpreter or apllication has to be deployed.
 See also
 --------
 
-* `SSH Daemon manpage<http://www.openbsd.org/cgi-bin/man.cgi/OpenBSD-current/man8/sshd.8>`_
-* `Ansible<http://docs.ansible.com>`_
+* `SSH Daemon manpage <http://www.openbsd.org/cgi-bin/man.cgi/OpenBSD-current/man8/sshd.8>`_
+* `Ansible <http://docs.ansible.com>`_
```

Comment:
ah, that is maybe because it's not all installed to a virtualenv + virtualenv active (see the developer docs).